resqpy/well/_trajectory.py

```diff
@@ -997,3 +997,5 @@ def __eq__(self, other):
         # TODO: more detailed equality comparison
         other_uuid = getattr(other, "uuid", None)
         return isinstance(other, self.__class__) and bu.matching_uuids(self.uuid, other_uuid)
+        other_uuid = getattr(other, "uuid", None)
```

Comment:
These two lines appear to be duplicates of the two above, and will never execute due to the earlier return statement.